cfme/tests/configure/test_default_views_cloud.py

```diff
@@ -1,25 +1,28 @@
 # -*- coding: utf-8 -*-
-
 import pytest
+
 from cfme import test_requirements
 from cfme.cloud.provider import CloudProvider
+from cfme.configure.settings import DefaultView
 from cfme.fixtures import pytest_selenium as sel
-import cfme.web_ui.toolbar as tb
-from cfme.web_ui import ButtonGroup, form_buttons, Quadicon, fill
+from cfme.web_ui import Quadicon, fill, toolbar as tb
+from utils import testgen
 from utils.appliance.implementations.ui import navigate_to
-from utils.providers import setup_a_provider as _setup_a_provider
-from cfme.configure import settings  # NOQA
-from cfme.cloud import instance  # NOQA
 
 pytestmark = [pytest.mark.tier(3),
-              test_requirements.settings]
+              test_requirements.settings,
+              pytest.mark.usefixtures('setup_provider')]
 
+pytest_generate_tests = testgen.generate(
```

Comment:
So this will *parametrize* the tests via the provider so we'll get at least one openstack and one ec2, currently we don't do this, we just add "a" provider. I think this is enough and that we shouldn't parametrize, thoughts?